catalog-rest-service/src/main/resources/json/schema/security/client/googleSSOClientConfig.json

```diff
@@ -6,9 +6,9 @@
   "type": "object",
   "javaType": "org.openmetadata.catalog.security.client.GoogleSSOClientConfig",
   "properties": {
-    "secretKey": {
-      "description": "Google SSO client secret key path or contents.",
-      "type": "string"
+    "credentials": {
+      "description": "Google SSO credentials values or credentials path.",
+      "$ref": "../credentials/gcsCredentials.json"
```

Comment:
This will need to be updated in the docs and shared with the users. Might need helm charts updating as well. Maybe we can tune it a bit to not make it break current installations